### ISSUE TITLE
Placeholder "EXTRA"

### ISSUE THREAD

**Issue by x1p** [Trophonix/TradePlus#51]
The placeholder `EXTRA` is translated in game with the word `economy`
Config:
```
extras:
  type:
    prefix: '&6&l!!&6> '
    empty: '&eHow much %EXTRA% to offer?'
    valid: '&aClick output slot to submit offer.'
    invalid: '&cInvalid amount entered!'
    maximum: '&cYou have %BALANCE% %EXTRA%'
  economy:
    enabled: true
    name: money
    material: gold_ingot
    display: '&eYour current money offer is &6%AMOUNT%'
    theirdisplay: '&eTheir current money offer is &6%AMOUNT%'
    lore:
    - '&fClick to edit your offer!'
    customModelData: 0
    increment: 10.0
    taxpercent: 0
    mode: type
```
When you trade and choose for a money offer, you are asked:
`empty: '&eHow much %EXTRA% to offer?'`
In game: `!!> How much economy to offer?`

Where does the word "economy" come from? I would assume this `EXTRA` would be translated into "money" because when you enter a large amount, it says: `!!> You have 100 money`. So it does know how to translate it (line: `maximum`)

**Comment by Trophonix** [comment]
Hey, thanks for reporting this! I think I know the issue

**Comment by Trophonix** [comment]
Fixed in 3.68
